sample/SampleHost/Program.cs

```diff
@@ -36,7 +36,7 @@ private static void CheckAndEnableAppInsights(JobHostConfiguration config)
             {
                 var filter = new LogCategoryFilter();
                 filter.DefaultLevel = LogLevel.Debug;
-                filter.CategoryLevels[LogCategories.Function] = LogLevel.Debug;
+                filter.CategoryLevels["Function"] = LogLevel.Debug;
```

Comment:
>["Function" [](start = 37, length = 11)

Switching from a symbol to a magical string literal seems like a step backwards... why the change?

Reply:
It's actually a bad sample; I'll fix it. It used to be that all user function logs came with "Function" and all executor logs came with "Host.Executor", but to make filtering nicer (replacing `TraceLevelAttribute`), you now can use the category "Function.Function1" to control a single function's logs. Or "Function.Function1.User" to control the user logs from a specific function. There are helpers for creating these categories: `LogCategories.CreateFunctionLog(categoryName)`. I'll use those here as a better example.